src/border.c

```diff
@@ -105,7 +105,9 @@ void border_resize(struct window *window)
     if (!window->border.id) return;
 
     CGRect frame = window_ax_frame(window);
-    if ((frame.size.width  == window->border.frame.size.width) &&
+    if ((frame.origin.x  == window->border.frame.origin.x) &&
+        (frame.origin.y == window->border.frame.origin.y) &&
```

Comment:
The (kinda) problem with this is that every time I've actually looked, it seems like `window->border.frame.origin` = `(x = 0, y = 0)`, so this is likely to always be `false`?

I suspect there is a deeper-in-the-(border struct, or sibling in window struct)  place the real position can be found, but I didn't find it (yet)

(I'd guess from the 0,0 that the border thinks its drawing relative to the window position, but ... clearly at some point it isn't really)